apps/segment/segment.js

```diff
@@ -1166,13 +1183,12 @@ async function showInfo() {
             td = row.insertCell();
             td.innerHTML = date;
             td = row.insertCell();
-            td.innerHTML = '<button class="btn-del" '+
-            'id=removeModel'+modelCount+' type="button"><i class="material-icons"'+
+            td.innerHTML = '<button class="btn btn-primary btn-xs my-xs-btn" '+
+            'id="removeModel" type="button"><i class="material-icons"'+
```

Comment:
This will bring back #361 . I think you don't have the latest ``` develop ``` branch . These changes were made in #362  and I think they shouldn't be changed back.

Reply:
Thanks for pointing out. I have fixed it.